content/terms/declarations-maintenance.en.md

```diff
@@ -110,7 +110,16 @@ The consensus is to consider that a service provider renaming a service (for exa
 ## How to get the `validUntil` date from an issue
 
 1. Go to the open issue and scroll to the last comment from `OTA-Bot`.
```

Comment:
Without a screenshot, I had to go to https://github.com/OpenTermsArchive/contrib-declarations/pull/1497#discussion_r1654952174 to understand the difference. What about:
```suggestion
1. Go to the open issue and scroll to the last action from `OTA-Bot` (comment or issue opening).
```

and adding a screenshot of the “reopening” case, similar to:

![](https://private-user-images.githubusercontent.com/364319/343197693-9e4f23bd-d5bc-45fd-8bdb-827aa7ddcaae.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk0MTQyMTcsIm5iZiI6MTcxOTQxMzkxNywicGF0aCI6Ii8zNjQzMTkvMzQzMTk3NjkzLTllNGYyM2JkLWQ1YmMtNDVmZC04YmRiLTgyN2FhN2RkY2FhZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyNlQxNDU4MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kY2Q4ODRiNjA4MjlmMTA0NzJlMzMzNmZjNTVmMWUxMzBjNzkyYWNiZjI5ZWVkYmZiYzJlOGU1YjgzOGFmNjZiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.wHbT7BQv1Ape95oJ-SZGVDmUhCKOoQM-ioCQbHq7cXU)